Casks/s/switch.rb

```diff
@@ -8,8 +8,8 @@
   homepage "https://www.nch.com.au/switch/"
 
   livecheck do
-    url :url
-    strategy :extract_plist
+    url "https://www.nch.com.au/switch/versions.html"
+    regex(%r{Version\s+v?(\d+(?:\.\d+)+)\s*</h2>\s*macOS\s+Release}i)
```

Comment:
```suggestion
    regex(%r{Version\s+v?(\d+(?:\.\d+)+)\s*</h\d>\s*macOS\s+Release}im)
```

A couple small tweaks:

* Using `</h\d>` instead of an explicit `</h2>` would allow this to continue working if they change the heading level in the future.
* Since we're matching loose text that can span multiple lines, we may want to use the `m` flag here. This currently works without it but sometimes regexes will stop matching when line breaks are added.